includes/Admin/Settings_Page_Assets.php

```diff
@@ -0,0 +1,131 @@
+<?php
+/**
+ * Handles asset enqueuing for the settings page.
+ *
+ * @package WordPress\AI
+ */
+
+namespace WordPress\AI\Admin;
+
+/**
+ * Manages scripts and styles for the admin settings page.
+ *
+ * @since 0.1.0
+ */
+class Settings_Page_Assets {
```

Comment:
Thinking about this more, I almost think we're better off having a generic assets loader as we'll need to load CSS and JS files for all other features, not just for the settings screen. I'd say we should repurpose this to support all asset loading